src/http/DataProduceEndpoints.js

```diff
@@ -82,6 +86,7 @@ module.exports = (streamFetcher, publisher, partitionFn = partition) => {
                 }
                 signatureType = req.query.signatureType ? parsePositiveInteger(req.query.signatureType) : 0
             } catch (err) {
+                logger.error(err.message)
```

Comment:
You sure you don't want to log the `err` or `err.stack`? Might find it tricky to debug without a stack trace.

Reply:
Updated